test/env/data.go

```diff
@@ -179,3 +184,79 @@ func BundleHasSyncedAllNamespaces(ctx context.Context, cl client.Client, name, e
 
 	return true
 }
```

Comment:
Can we remove BundleHasSyncedAllNamespaces and use EventuallyBundleHasSyncedAllNamespaces instead everywhere?

Reply:
Yeah, makes sense to do that here!